### ISSUE TITLE
One question

### ISSUE THREAD

**Issue by wanlmq6165163** [MCG-NJU/TDN#19]
Hi！
I tested sthv1 on two GTX Titan X as follows:
python -m torch.distributed.launch --master_port 12347 --nproc_per_node=2 \
            main.py  something  RGB --arch resnet50 --num_segments 8 --gd 20 --lr 0.005 \
            --lr_scheduler step --lr_steps  30 45 55 --epochs 60 --batch-size 16 \
            --wd 5e-4 --dropout 0.5 --consensus_type=avg --eval-freq=1 -j 4 --npb 
But the accuracy in modelzoom is not obtained,
In addition, I tried to implement the core code of TDN on the basis of TSM in mmaction2, and only achieved 48.9%. 
What details did I ignore?
Look forward to your reply！

**Comment by yztongzhan** [comment]
Hi @wanlmq6165163! We use 8 GPUs for training, so I didn't konw if it is possible to reproduce our results only using 2 GPUs.
However, I have some tips for you:
1. You set `batch-size 16` for two GPUs which means your mini-batch is `32`, and `learning rate = 0.005` which I think is too small.
> I tried to implement the core code of TDN on the basis of TSM in mmaction2
2. We use **flip aug** for SSV1 training,  which also support in [MMaction2 for such TSM](https://github.com/open-mmlab/mmaction2/blob/master/configs/recognition/tsm/tsm_r50_flip_1x1x8_50e_sthv1_rgb.py). But we want to point out that our `label generation script` is different from MMaction2, you need to check carefully.

**Comment by yztongzhan** [comment]
Hi @wanlmq6165163! For your convenience, we provide the [training log](https://drive.google.com/file/d/18Pfiw-f-hyaQ9F_R3TYH8nwu0W6g2Mj_/view?usp=sharing) and [tensorboard file](https://drive.google.com/file/d/1qjrZtgRv9HuNmNI2GDiBTVZwIFjxaG4E/view?usp=sharing) for SSV1 TDN-50. We hope these could be helpful for you.

**Comment by wanlmq6165163** [comment]
Do you mean the more GPUs, the better? I wonder if there is any relevant literature to prove this, and, how to set the learning rate for my situation..

**Comment by yztongzhan** [comment]
I mean your  `batch size` is not enough. You can refer [here](https://arxiv.org/abs/1706.02677). Thanks!

**Comment by wanlmq6165163** [comment]
I got it！
Thanks for your help！
Looking forward to your new works!

**Comment by yztongzhan** [comment]
@wanlmq6165163 Thanks so much for sharing your interest in our TDN!
I you don't have more questions, I wil close this issue.

**Comment by wanlmq6165163** [comment]
Well! Turn it off!
